prog/dftb+/lib_dftb/dftd4param.F90

```diff
@@ -83,48 +91,51 @@ module dftbp_dftd4param
   integer, parameter :: imagFrequencies = 23
 
 
-  !> Damping parameters for DFT-D4 calculation.
+  !> Damping parameters for DFT-D4 calculation
   type :: TDispDftD4Inp
 
-    !> Scaling parameter for dipole-dipole coefficients.
+    !> Scaling parameter for dipole-dipole coefficients
     real(dp) :: s6 = 1.0_dp
 
-    !> Scaling parameter for dipole-quadrupole coefficients.
+    !> Scaling parameter for dipole-quadrupole coefficients
     real(dp) :: s8
 
-    !> Scaling parameter for quadrupole-quadrupole coefficients.
+    !> Scaling parameter for quadrupole-quadrupole coefficients
     real(dp) :: s10 = 0.0_dp
 
-    !> Scaling parameter for non-additive triple dipole coefficients.
+    !> Scaling parameter for non-additive triple dipole coefficients
     real(dp) :: s9
 
-    !> Scaling parameter for <r4>/<r2> expectation value based critical radii.
+    !> Scaling parameter for <r4>/<r2> expectation value based critical radii
     real(dp) :: a1
 
-    !> Constant offset of critical radii.
+    !> Constant offset of critical radii
     real(dp) :: a2
 
     !> Exponent of for the zero-damping function used for non-addititive triple dipole
-    !> contributions.
+    !> contributions
     real(dp) :: alpha = 16.0_dp
 
-    !> Cutoff radius for dispersion interactions.
+    !> Cutoff radius for dispersion interactions
     real(dp) :: cutoffInter = 64.0_dp
 
-    !> Cutoff radius for three-body interactions.
+    !> Cutoff radius for three-body interactions
     real(dp) :: cutoffThree = 40.0_dp
 
-    !> Gaussian weighting factor for interpolation of dispersion coefficients.
+    !> Gaussian weighting factor for interpolation of dispersion coefficients
     real(dp) :: weightingFactor = 6.0_dp
 
-    !> Maximum charge scaling height for partial charge extrapolation.
+    !> Maximum charge scaling height for partial charge extrapolation
     real(dp) :: chargeScale = 3.0_dp
 
-    !> Charge scaling steepness for partial charge extrapolation.
+    !> Charge scaling steepness for partial charge extrapolation
     real(dp) :: chargeSteepness = 2.0_dp
 
+    !> Dispersion is evaluated self-consistent
+    logical :: tSelfConsistent = .false.
```

Comment:
You don't have to prefix with `t` for logicals, could also use the more natural `selfConsistent` (or `isSelfConsistent`) instead.